core/invoker/src/main/scala/whisk/core/containerpool/ContainerPool.scala

```diff
@@ -71,6 +71,8 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
   var freePool = immutable.Map.empty[ActorRef, ContainerData]
   var busyPool = immutable.Map.empty[ActorRef, ContainerData]
   var prewarmedPool = immutable.Map.empty[ActorRef, ContainerData]
+  var lastRetryLog: Option[Long] = None
+  val logMessageInterval = 10000;
```

Comment:
Can we use `FiniteDuration` here and work with higher level abstractions (`Instant`) instead of `Long`s?

Reply:
I changed the logic to compare a given `Deadline`